tests/test_benchmark.py

```diff
@@ -116,7 +116,7 @@ def __init__(self, name, cc, cxx, args=[OPTIMIZATIONS]):
   def build(self, parent, filename, args, shared_args, emcc_args, native_args, native_exec, lib_builder, has_output_parser):
     self.parent = parent
     if lib_builder:
-      native_args += lib_builder(self.name, native=True, env_init={'CC': self.cc, 'CXX': self.cxx})
+      native_args += lib_builder(self.name, native=True, env_init={'CC': self.cc, 'CXX': self.cxx, 'CXXFLAGS': "-Wno-c++11-narrowing"})
```

Comment:
why is this change necessary - it's for native, no cheerp?

Reply:
As far as I can see the benchmarking system uses fastcomp's clang to build the native test and with that bullet won't compile without the ```-Wno-c++11-narrowing``` flag.

Here is what version of fastcomp I am using

```
$ emsdk/clang/fastcomp/build_incoming_64/bin/clang++ -v
clang version 6.0.1 (https://github.com/kripken/emscripten-fastcomp-clang.git 7feb22c20d97547e66d14e6265a5b7274f8b96a9) (https://github.com/kripken/emscripten-fastcomp.git 2a333e6b14e94de7fa32dd0a772886d028515708) (emscripten 1.38.25 : 1.38.25)
```

The fact that the flag is required is surprising to me as well. Is the bullet benchmark regularly build?

Reply:
Thanks - I'd guess what's going on is that part of the bullet configure script is sensitive to the local system (i.e. it uses the system compiler in some cases, ignoring CC/CXX).  The benchmarks run on my machines, so your system compiler may be older/newer. Anyhow, seems fine to add.